ReactAndroid/src/main/third-party/android/support-annotations/BUCK

```diff
@@ -1,14 +1,10 @@
 load("//tools/build_defs:fb_native_wrapper.bzl", "fb_native")
-load("//tools/build_defs/oss:rn_defs.bzl", "rn_prebuilt_jar")
+load("//tools/build_defs/oss:rn_defs.bzl", "react_native_dep")
```

Comment:
Can you revert changes in this file and upgrade version of support library from 27.1.1 to 28.0.0 in the original file?